### DISCUSSION TITLE
does eigr/spawn support function in elixir ????

### DISCUSSION THREAD

**Discussion by Lautarosg** [eigr/spawn#114] [Q&A] [answered]
hi dears, i want to know if can write function in elixir/erlang over eigr

**Comment by sleipnir** [comment, marked as answer]
> hi dears, i want to know if can write function in elixir/erlang over eigr

Hello @Lautarosg. Thanks for the question.
Yes you can. We have an SDK available for Elixir, I'll leave links to documentation and sample applications below. Just noting that as we are still in the pre-launch phase, if you encounter any problems, we ask you to open an issue so that we can deal with it.

https://github.com/eigr/spawn/blob/main/spawn_sdk/spawn_sdk/README.md
https://hexdocs.pm/spawn_sdk/readme.html

https://github.com/eigr/spawn#examples

Ps.: You can also contact us through our Discord channel https://discord.gg/JRBddsmG
